official_scenario_configs/cifar10_baseline.json

```diff
@@ -18,15 +18,15 @@
     "metric": {
         "means": true,
         "perturbation": "linf",
-        "record_metric_per_sample": true,
+        "record_metric_per_sample": false,
         "task": [
             "categorical_accuracy"
         ]
     },
     "model": {
         "fit": true,
         "fit_kwargs": {
-            "nb_epochs": 5
+            "nb_epochs": 20
```

Comment:
So I lowered this to 5 since there is no logging for pytorch fit_generator in ART (will make that issue later today). For the demo I think we'll want this to be a smaller number?